sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactFacadeEjb.java

```diff
@@ -213,6 +215,8 @@
 	@EJB
 	private ConfigFacadeEjbLocal configFacade;
 	@EJB
+	private ContactService contactService;
```

Comment:
please remove this Ejb injection, the service is already injected via constructor injection
do so for all the other core facade ejbs

Reply:
removed